### PR TITLE
btrfs-device: add page

### PR DESCRIPTION
**Pull request by sbrl**
Another one for #5103

### CODE REVIEW

File: pages/linux/btrfs-device.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# btrfs device
+
+> Manage devices in a btrfs filesystem.
+> More information: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-device>.
+
+- Add one or more devices to a btrfs filesystem:
+
+`sudo btrfs device add {{path/to/block_device}} [{{...}}] {{path/to/btrfs_filesystem}}`
```
Comment (bl-ue) [inline pages/linux/btrfs-device.md:8]: @sbrl is this official tldr-pages syntax for an arbitrary number of parameters?
What about something like this:
```suggestion
`sudo btrfs device add {{path/to/block_device1}} {{path/to/block_device2}} {{path/to/btrfs_filesystem}}`
```
?

Comment (sbrl) [reply pages/linux/btrfs-device.md:8]: I think I've seen it done somewhere before, but I can't remember.

Comment (bl-ue) [reply pages/linux/btrfs-device.md:8]: I like my proposal.  Can we do it?

Comment (sbrl) [reply pages/linux/btrfs-device.md:8]: I'd suggest enclosing the 2nd block device in square brackets to indicate it's optional. We have done that elsewhere.

Comment (bl-ue) [reply pages/linux/btrfs-device.md:8]: I've updated my comment.

**Review by bl-ue — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**